packages/components/src/stories/VsCodeExtension.stories.js

```diff
@@ -36,9 +36,10 @@ export default {
       },
       defaultValue: 'default',
     },
-  },
-  args: {
-    appMapUploadable: true,
+    shareEnabled: {
+      control: 'boolean',
+      defaultValue: 'true',
```

Comment:
This declares the control as a boolean but sets the default value to a string. Is this right?